test/runtests.jl

```diff
@@ -1,7 +1,14 @@
 using LinearAlgebra: length
 using LinearAlgebra, Random, Test
 using ProximalOperators
-using ADNLPModels, OptimizationProblems, OptimizationProblems.ADNLPProblems, NLPModels, NLPModelsModifiers, RegularizedProblems, RegularizedOptimization, SolverCore
+using ADNLPModels,
+  OptimizationProblems,
+  OptimizationProblems.ADNLPProblems,
+  NLPModels,
+  NLPModelsModifiers,
+  RegularizedProblems,
+  RegularizedOptimization,
+  SolverCore
```

Comment:
@MaxenceGollier @MohamedLaghdafHABIBOULLAH Please configure your formatter locally. These changes should already have been part of the PR.